ground/openpilotgcs/src/plugins/setupwizard/vehicletemplateexportdialog.cpp

```diff
@@ -50,7 +50,7 @@
 
 const char *VehicleTemplateExportDialog::EXPORT_FIXEDWING_NAME = "fixedwing";
 const char *VehicleTemplateExportDialog::EXPORT_MULTI_NAME     = "multirotor";
-const char *VehicleTemplateExportDialog::EXPORT_HELI_NAME      = "helicopter";
+const char *VehicleTemplateExportDialog::EXPORT_HELI_NAME = "helicopter";
```

Comment:
Changing the uncrustify config with:
```
diff --git a/make/uncrustify/uncrustify.cfg b/make/uncrustify/uncrustify.cfg
index 912b82b..d2c2c8d 100644
--- a/make/uncrustify/uncrustify.cfg
+++ b/make/uncrustify/uncrustify.cfg
@@ -765,10 +765,10 @@ align_var_def_attribute                  = false    # false/true
 align_var_def_inline                     = true     # false/true

 # The span for aligning on '=' in assignments (0=don't align)
-align_assign_span                        = 3        # number
+align_assign_span                        = 1        # number

 # The threshold for aligning on '=' in assignments (0=no limit)
-align_assign_thresh                      = 4        # number
+align_assign_thresh                      = 0        # number

 # The span for aligning on '=' in enums (0=don't align)
 align_enum_equ_span                      = 3        # number
```
Gives better alignment, IMHO:

```
diff --git a/ground/openpilotgcs/src/plugins/setupwizard/vehicletemplateexportdialog.cpp b/ground/openpilotgcs/src/plugins/setupwizard/vehicletemplateexportdialog.cpp
index 301a49e..563ddfc 100644
--- a/ground/openpilotgcs/src/plugins/setupwizard/vehicletemplateexportdialog.cpp
+++ b/ground/openpilotgcs/src/plugins/setupwizard/vehicletemplateexportdialog.cpp
@@ -50,7 +50,7 @@

 const char *VehicleTemplateExportDialog::EXPORT_FIXEDWING_NAME = "fixedwing";
 const char *VehicleTemplateExportDialog::EXPORT_MULTI_NAME     = "multirotor";
-const char *VehicleTemplateExportDialog::EXPORT_HELI_NAME = "helicopter";
+const char *VehicleTemplateExportDialog::EXPORT_HELI_NAME      = "helicopter";
 const char *VehicleTemplateExportDialog::EXPORT_SURFACE_NAME   = "surface";
 const char *VehicleTemplateExportDialog::EXPORT_CUSTOM_NAME    = "custom";

@@ -216,7 +216,7 @@ void VehicleTemplateExportDialog::saveTemplate(QString path)
     exportObject["controller"] = ui->Controllers->currentText();
     exportObject["comment"]    = ui->Comment->document()->toPlainText();
     QUuid uuid = QUuid::createUuid();
-    exportObject["uuid"]       = uuid.toString();
+    exportObject["uuid"] = uuid.toString();

     if (!m_image.isNull()) {
         QByteArray bytes;
@@ -231,11 +231,11 @@ void VehicleTemplateExportDialog::saveTemplate(QString path)

     const char *fileType = ".optmpl";

-    QString fileName     = QString("%1-%2-%3%4")
-                           .arg(fixFilenameString(ui->Name->text(), 20))
-                           .arg(fixFilenameString(ui->Type->text(), 30))
-                           .arg(fixFilenameString(uuid.toString().right(12)))
-                           .arg(fileType);
+    QString fileName = QString("%1-%2-%3%4")
+                       .arg(fixFilenameString(ui->Name->text(), 20))
+                       .arg(fixFilenameString(ui->Type->text(), 30))
+                       .arg(fixFilenameString(uuid.toString().right(12)))
+                       .arg(fileType);

     QString fullPath;
     if (path.isEmpty()) {
```
Setting `align_assign_span` to `1`changes uncrustify to only align consecutive lines. So, adding a blank line, or an non-assignment statement breaks the alignment block, it seems.

I prefer this much more than what we have today.